AlmesCore/README.md

```diff
@@ -0,0 +1,9 @@
+# Redirects for the Almes Core Metadata Schema
+
+## Maintainers:
+- GitHub ID: @AlmesCore
+- GitHub ID: @Filipi-Soares
```

Comment:
Just to check, `Filipi-Soares` is the same as `filipisoares` that did squash commit?

Reply:
Yes, it's me. Sorry for this. I used cmd to squash the commit and forgot to change my user name.